### PR TITLE
docs: added gitlab and jenkins cicd docs

### PR DESCRIPTION
**Pull request by emamihe**
## Pull request description 
added the docs for gitlab ci/cd and jenkins

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

**Comment by vsukhin** [comment]
gj, Hamid!

File: docs/docs/articles/jenkins.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+This Jenkins integration offers a versatile solution for managing your pipeline workflows and is compatible with Testkube Cloud, Testkube Enterprise, and the open-source Testkube platform. It allows Jenkins users to effectively utilize Testkube's capabilities within their CI/CD pipelines, providing a robust and flexible framework for test execution and automation.
+
+## Testkube Cloud
+
+### How to configure Testkube CLI action for TK Cloud and run a test
+
+To use this Jenkins CI/CD for the [Testkube Cloud](https://cloud.testkube.io/), you need to create [API token](https://docs.testkube.io/testkube-cloud/articles/organization-management/#api-tokens).
+Then, pass the **organization** and **environment** IDs, along with the **token** and other parameters specific for your use case.
+
+If test is already created, you may directly run it using the command `testkube run test test-name -f` . However, if you need to create a test in this workflow, please add a creation command, e.g.: `testkube create test --name test-name --file path_to_file.json`.
```
Comment (ypoplavs) [inline docs/docs/articles/jenkins.md:13]: We can also install Testkube via UI so I think we can say here that we can use both Jenkinsfile (say a few words about it and point to official documentation) with declarative pipeline and Classic UI.

**Review by nicufk — APPROVED** [review]
looks good, it will need @jfermi approval

File: docs/docs/articles/gitlab.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# Testkube Gitlab CI
+
+The Testkube GitLab CI/CD integration facilitates the installation of Testkube and allows the execution of any [Testkube CLI](https://docs.testkube.io/cli/testkube) command within a GitLab CI/CD pipeline. This integration can be seamlessly incorporated into your GitLab repositories to enhance your CI/CD workflows.
+The integration offers a versatile approach to align with your pipeline requirements and is compatible with Testkube Cloud, Testkube Enterprise, and the open-source Testkube platform. It enables GitLab users to leverage the powerful features of Testkube directly within their CI/CD pipelines, ensuring efficient and flexible test execution.
+
+## Testkube Cloud
+
+### How to configure Testkube CLI action for TK Cloud and run a test
+
+To use this Gitlab CI for the [Testkube Cloud](https://cloud.testkube.io/), you need to create [API token](https://docs.testkube.io/testkube-cloud/articles/organization-management/#api-tokens).
```
Comment (jfermi) [inline docs/docs/articles/gitlab.md:10]:
```suggestion
To use this Gitlab CI for [Testkube Cloud](https://cloud.testkube.io/), you need to create [API tokens](https://docs.testkube.io/testkube-cloud/articles/organization-management/#api-tokens).
```

File: docs/docs/articles/gitlab.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+Then, pass the **organization** and **environment** IDs, along with the **token** and other parameters specific for your use case.
+
+If test is already created, you may directly run it using the command `testkube run test test-name -f` . However, if you need to create a test in this workflow, please add a creation command, e.g.: `testkube create test --name test-name --file path_to_file.json`.
```
Comment (jfermi) [inline docs/docs/articles/gitlab.md:13]:
```suggestion
If a test is already created, you can run it using the command `testkube run test test-name -f` . However, if you need to create a test in this workflow, please add a creation command, e.g.: `testkube create test --name test-name --file path_to_file.json`.
```

File: docs/docs/articles/gitlab.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  stage: test
+  script:
+    - echo "Running Testkube test..."
+    - testkube run test test-name -f
+ ```
+## Testkube OSS
+
+### How to configure Testkube CLI action for TK OSS and run a test
+
+To connect to the self-hosted instance, you need to have **kubectl** configured for accessing your Kubernetes cluster, and simply passing optional namespace, if Testkube is not deployed in the default **testkube** namespace.
```
Comment (jfermi) [inline docs/docs/articles/gitlab.md:63]:
```suggestion
To connect to the self-hosted instance, you need to have **kubectl** configured for accessing your Kubernetes cluster, and pass an optional namespace, if Testkube is not deployed in the default **testkube** namespace. 
```

File: docs/docs/articles/gitlab.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    - echo "Running Testkube test..."
+    - testkube run test test-name -f
+ ```
+## Testkube OSS
+
+### How to configure Testkube CLI action for TK OSS and run a test
+
+To connect to the self-hosted instance, you need to have **kubectl** configured for accessing your Kubernetes cluster, and simply passing optional namespace, if Testkube is not deployed in the default **testkube** namespace. 
+
+If test is already created, you may directly run it using the command `testkube run test test-name -f` . However, if you need to create a test in this workflow, please add a creation command, e.g.: `testkube create test --name test-name --file path_to_file.json`.
```
Comment (jfermi) [inline docs/docs/articles/gitlab.md:65]:
```suggestion
If a test is already created, you can run it using the command `testkube run test test-name -f` . However, if you need to create a test in this workflow, please add a creation command, e.g.: `testkube create test --name test-name --file path_to_file.json`.
```

File: docs/docs/articles/gitlab.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    - testkube cloud init --namespace $NAMESPACE
+
+run-testkube-test:
+  stage: test
+  script:
+    - echo "Running Testkube test..."
+    - testkube run test test-name -f
+```
+
+Steps to connect to your Kubernetes cluster differ for each provider. You should check the docs of your Cloud provider on how to connect to the Kubernetes cluster from Gitlab CI.
```
Comment (jfermi) [inline docs/docs/articles/gitlab.md:89]:
```suggestion
The steps to connect to your Kubernetes cluster differ for each provider. You should check the docs of your Cloud provider for how to connect to the Kubernetes cluster from Gitlab CI.
```

File: docs/docs/articles/jenkins.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
+# Testkube Jenkins
+
+The Testkube Jenkins integration streamlines the installation of Testkube, enabling the execution of any [Testkube CLI](https://docs.testkube.io/cli/testkube) command within Jenkins pipelines. This integration can be effortlessly integrated into your Jenkins setup, enhancing your continuous integration and delivery processes.
+This Jenkins integration offers a versatile solution for managing your pipeline workflows and is compatible with Testkube Cloud, Testkube Enterprise, and the open-source Testkube platform. It allows Jenkins users to effectively utilize Testkube's capabilities within their CI/CD pipelines, providing a robust and flexible framework for test execution and automation.
+
+## Testkube Cloud
+
+### How to configure Testkube CLI action for TK Cloud and run a test
+
+To use this Jenkins CI/CD for the [Testkube Cloud](https://cloud.testkube.io/), you need to create [API token](https://docs.testkube.io/testkube-cloud/articles/organization-management/#api-tokens).
```
Comment (jfermi) [inline docs/docs/articles/jenkins.md:10]:
```suggestion
To use Jenkins CI/CD for [Testkube Cloud](https://cloud.testkube.io/), you need to create [API tokens](https://docs.testkube.io/testkube-cloud/articles/organization-management/#api-tokens).
```

File: docs/docs/articles/jenkins.md (hunk 1 above)
Comment (jfermi) [inline docs/docs/articles/jenkins.md:13]:
```suggestion
If a test is already created, you can run it using the command `testkube run test test-name -f` . However, if you need to create a test in this workflow, please add a creation command, e.g.: `testkube create test --name test-name --file path_to_file.json`.
```

File: docs/docs/articles/jenkins.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+    }
+}
+
+```
+
+## Testkube OSS
+
+### How to configure Testkube CLI action for TK OSS and run a test
+
+To connect to the self-hosted instance, you need to have **kubectl** configured for accessing your Kubernetes cluster, and simply passing optional namespace, if Testkube is not deployed in the default **testkube** namespace.
```
Comment (jfermi) [inline docs/docs/articles/jenkins.md:52]:
```suggestion
To connect to the self-hosted instance, you need to have **kubectl** configured for accessing your Kubernetes cluster, and pass an optional namespace, if Testkube is not deployed in the default **testkube** namespace. 
```

File: docs/docs/articles/jenkins.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+```
+
+## Testkube OSS
+
+### How to configure Testkube CLI action for TK OSS and run a test
+
+To connect to the self-hosted instance, you need to have **kubectl** configured for accessing your Kubernetes cluster, and simply passing optional namespace, if Testkube is not deployed in the default **testkube** namespace. 
+
+If test is already created, you may directly run it using the command `testkube run test test-name -f` . However, if you need to create a test in this workflow, please add a creation command, e.g.: `testkube create test --name test-name --file path_to_file.json`.
```
Comment (jfermi) [inline docs/docs/articles/jenkins.md:54]:
```suggestion
If a test is already created, you can run it using the command `testkube run test test-name -f` . However, if you need to create a test in this workflow, please add a creation command, e.g.: `testkube create test --name test-name --file path_to_file.json`.
```

File: docs/docs/articles/jenkins.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+            steps {
+                // Run a Testkube test
+                sh 'testkube run test test-name -f'
+            }
+        }
+    }
+}
+```
+
+Steps to connect to your Kubernetes cluster differ for each provider. You should check the docs of your Cloud provider on how to connect to the Kubernetes cluster from Jenkins CI/CD
```
Comment (jfermi) [inline docs/docs/articles/jenkins.md:86]:
```suggestion
The steps to connect to your Kubernetes cluster differ for each provider. You should check the docs of your Cloud provider for how to connect to the Kubernetes cluster from Jenkins CI/CD
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Pull request merged**